src/main/java/uk/gov/hmcts/reform/ucmc/service/FeesService.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.reform.ucmc.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fees.client.FeesClient;
+import uk.gov.hmcts.reform.fees.client.model.FeeLookupResponseDto;
+import uk.gov.hmcts.reform.payments.client.models.FeeDto;
+import uk.gov.hmcts.reform.ucmc.config.FeesConfiguration;
+import uk.gov.hmcts.reform.ucmc.model.ClaimValue;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.math.RoundingMode;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
```

Comment:
Note: A constructor doesn't have to be annotated with `@Autowired` as long as there is only one in a component and Spring can unambiguously choose it as the right one to instantiate a new object.
```suggestion
@RequiredArgsConstructor
```

Reply:
didn't know that 👍 but I have 2 components injected now

Reply:
I meant one constructor in a component, not one component injected.